server/autotest_remote.py

```diff
@@ -239,6 +239,12 @@ def _install_using_send_file(self, host, autodir):
             commands.append("touch '%s'/__init__.py" % abs_path)
         host.run(';'.join(commands))
 
+        extra_src_dir = "%s/tests" % self.source_material
+        light_files = [os.path.join(extra_src_dir, f)
+                       for f in os.listdir(extra_src_dir)
+                       if f != "virt"]
+        host.send_file(light_files, "%s/tests" % autodir, delete_dest=True)
```

Comment:
Hi @spcui there should be make an optional choice for user. Sometimes it will cause extra work to sync useless tests.